docs/src/pages/system/the-sx-prop/Example.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import TrendingUpIcon from '@material-ui/icons/TrendingUp';
+import Box from '@material-ui/core/Box';
```

Comment:
I assume to be consistent with the section? But not sure

```suggestion
import Box from '@material-ui/system/Box';
```